observability/cockpit/api-cli/configuring-grafana-agent.mdx

```diff
@@ -0,0 +1,178 @@
+        sysfs_path: /host/sys
+        procfs_path: /host/proc
+    ```
+    Your configuration file should look like the following:
+
+    ```
+    metrics:
+    wal_directory: /tmp/agent
+    global:
+        scrape_interval: 15s
+        remote_write:
+        - url: https://metrics.prd.obs.fr-par.scw.cloud/api/v1/push
+        headers:
+            "X-Token": 4zz-3cGLS5tzuDRb6LkEdpem7eOqj8EFecfWcJmTvrakS15gu07iEvtC5RfYDzi8
+
+
+    logs:
+    positions_directory: /tmp/positions/
+    configs:
+    - name: scw
+        clients:
+        - url: https://logs.prd.obs.fr-par.scw.cloud/loki/api/v1/push
+        tenant_id: 4zz-3cGLS5tzuDRb6LkEdpem7eOqj8EFecfWcJmTvrakS15gu07iEvtC5RfYDzi8
+        scrape_configs:
+        - job_name: varlogs
+        static_configs:
+        - targets: [localhost]
+            labels:
+            job: varlogs
+            __path__: /host/root/var/log/*.log
+
+
+    integrations:
+    node_exporter:
+        enabled: true
+        rootfs_path: /host/root
+        sysfs_path: /host/sys
+        procfs_path: /host/proc
+    ```
+6. In the same folder, create another file and name it `docker-compose.yaml`.
+7. Copy the following template and paste it into your file:
+
+    ```
+    version: "3"
+    services:
+    agent:
+        image: grafana/agent:latest
+        restart: always
+        volumes:
+        - "./config.yaml:/etc/agent-config/agent.yaml:ro,rslave"
+        - "/:/host/root:ro,rslave"
+        - "/sys:/host/sys:ro,rslave"
+        - "/proc:/host/proc:ro,rslave"
+        - "/tmp/agent:/etc/agent"
+        entrypoint:
+        - /bin/agent
+        - -config.file=/etc/agent-config/agent.yaml
+        - -metrics.wal-directory=/tmp/agent/wal
+        network_mode: "host"
+        pid: "host"
+    ```
+    <Message type="important">
+     If you are using OSX, remove all occurrences of `,rslave` in your configuration file.
+    </Message>
+
+8. Run the following command to open your folder containing your configuration files:
+
+    ```
+    cd <your folder>
+    ```
+9. Run the following command to bring the Grafana Agent container up:
+
+    ```
+    docker-compose up
+    ```
+
+## Visualizing metrics on Grafana
+
+1. [Log into Grafana](/observability/cockpit/how-to/access-grafana-and-managed-dashboards/) using your [Grafana credentials](/observability/cockpit/how-to/retrieve-grafana-credentials/).
+2. Click the **Dashboards** icon, then click **Browse**.
+    <Lightbox src="scaleway_obs_dashboards_browse.webp" alt="" />
+3. Click **Import** on the top right corner of your screen to import your Grafana dashboard.
+4. Enter `1860`in the ID field and click **Load**.
+    <Lightbox src="scaleway_obs_import_dashboard.webp" alt="" />
+5. Select the **Scaleway Metrics Prometheus** instance as the data source and click **Import**. Your dashboard displays with your metrics.
+
+## Visualizing logs on Grafana
+
+<Message type="important">
+ It is not currently possible to visualize logs if you are using OSX.
```

Comment:
It is not possible to collect logs, visualization works fine maybe this note should be added in the grafana agent configuration.